class AuthError(Exception):
    pass


class SignatureError(AuthError):
    pass
